src/BlazorUI/Bit.BlazorUI/Components/Buttons/ButtonGroup/BitButtonGroup.scss

```
@@ -62,28 +81,169 @@
 }
 
+.bit-btg-bdg {
+    display: flex;
+    flex: none;
+    line-height: 1;
+    align-items: center;
+    justify-content: center;
+    padding: spacing(0.25) spacing(0.75);
+    border-radius: 9999px;
+    font-size: calc(var(--bit-btg-itm-fontsize) * 0.8);
+    // Tinting the current foreground keeps the badge legible on every variant,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)